### PR TITLE
Adjustments

### PR DESCRIPTION
**Pull request by VespertinePetrichor**
## Additions
+ Corrected Path
+ Added extensions
+ Added track priority
+ Added is_valid conditions
  - Alliance
  - Horde
  - War Tracks [WC2]
  - Cultural Tracks [WC3]
## Changes
- Removed calls (they were causing errors and vanilla tracks to play)
- Removed Volume (was not a recognised effect)
---
### Testing
Check error log for 'music'
Test tracks using `Music.playback [track name]`, as defined in `music.txt`

### CODE REVIEW

**Review by Gaben1025 — APPROVED** [review]
LGTM

**Comment by Gaben1025** [comment]
All tracks work fine and there are no errors

**Comment by ercarp** [comment]
Not sure what I'm doing wrong, but when I try to start up the game with this sub-mod enabled, it crashes on start-up without even giving the crash pop-up. Tried to change load order of the mods and tested in both `1.5.0.1` and `1.5.0.2`. Also tested on both the `main` and `Adjustments` branch and it still happens.

File: Warcraft-Guardians-of-Azeroth-2-Music-Submod.mod [hunk 1]
```diff
@@ -4,4 +4,4 @@ tags={
 }
 name="Warcraft Guardians of Azeroth 2 Music Submod"
 supported_version="1.5.0.2"
-path="C:/Users/akozh/Documents/Paradox Interactive/Crusader Kings III/mod/Warcraft-Guardians-of-Azeroth-2-Music-Submod"
\ No newline at end of file
+path="/mod/Warcraft-Guardians-of-Azeroth-2-Music-Submod"
```
Comment (ercarp) [inline Warcraft-Guardians-of-Azeroth-2-Music-Submod.mod:7]: The extra / at the start of the mod directory was causing problems for me. I had to delete it for GoA2 launcher to detect the mod properly.

I suggest changing this to `path="mod/Warcraft-Guardians-of-Azeroth-2-Music-Submod"`

**Review by ercarp — COMMENTED** [review]
Went through all the soundtracks and I can confirm they work perfectly fine. I also like the soundtracks you've picked and I think a lot of players will appreciate having some WC2 soundtracks in the mix as well.

**EDIT**: Also, almost forgot to mention this, but someone on the server requested for [Nightsong](https://www.youtube.com/watch?v=HEK4OXIx9NE) to be included in the Music Submod. It's a very iconic Warcraft OST so adding it as a music track for Night Elves would no doubt be appreciated by many. But that one's just a suggestion.

**Comment by nsYW** [comment]
**Work fine for me!** But I think some very explosive bgm need a fade-in (like stormwind and ironforge), otherwise it might scare people.

By the way, I suggest add the following musics :
    1. Nightsong - Death of Ysera extended ver (https://www.youtube.com/watch?v=ETPjddfrk_w)
    2. Medivh main theme (https://www.youtube.com/watch?v=qR6T3hvwxSE) 
    3. Way Of The Monk (https://www.youtube.com/watch?v=2g-wBrU9keE)
    4. Why Do We Fight? - Cinematic Music (https://www.youtube.com/watch?v=RZaSnfxWsBg&t)
    5. Heart Of Pandaria (timecode: 5:14 ~ end) (https://www.youtube.com/watch?v=MsDRarTOtas)

**Pull request merged**